helm/microcosm/templates/osm2pgsql-statefulset.yaml

```diff
@@ -28,7 +28,7 @@ spec:
         {{- if eq .Values.cloudProvider "minikube" }} 
         image: "{{ .Values.osm2pgsql.image.minikube }}"
         {{- end }}
-        command: ['./start.sh']
```

Comment:
let's stick for `start.py` for all scripts

Reply:
fixed